common/network-common/src/main/java/org/apache/spark/network/util/TransportConf.java

```diff
@@ -257,4 +257,7 @@ public Properties cryptoConf() {
     return CryptoUtils.toCryptoConf("spark.network.crypto.config.", conf.getAll());
   }
 
+  public long maxChunksBeingTransferred() {
+    return conf.getLong("spark.network.shuffle.maxChunksBeingTransferred", Long.MAX_VALUE);
```

Comment:
This default value totally depends on the JVM heap size. Seems hard to pick up a reasonable value. If it's too small, then if a user uses a large heap size, when they upgrade, their shuffle service may start to fail. If it's too large, it's just the same as MAX_VALUE.